src/stravalib/strava_model.py

```diff
@@ -4,7 +4,7 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import Literal, Optional
+from typing import Dict, List, Literal, Optional
```

Comment:
i see what you're saying with it reverting to the typing List / Dict vs the native to Python list dict objects.  And i see the docstring typo as well. 

i think the good news here is that none if this should impact how the code runs right?